docs/6.x/pack.md

```diff
@@ -752,16 +751,25 @@ example/
 
 When building the Cluster Image, the `tele build` command will find
 directories with Helm charts (determined by the presence of `Chart.yaml` file)
-and vendor all Docker images they reference into the resulting installer
-tarball.
+and vendor all Docker images they reference into the resulting image tarball.
 
-!!! note:
-    The machine running `tele build` must have Helm binary [installed](https://docs.helm.sh/using_helm/#installing-helm)
-    and available in PATH as well as its [template plugin](https://docs.helm.sh/using_helm/#installing-a-plugin).
+The `tele build` command also allows to override Helm chart values at build time
+via `--values` and `--set` flags. These values will be taken into account when
+rendering Helm templates which is useful if you need to vendor a specific version
+of a certain Docker image or pull it from a specific registry.
+
+The flags have the same meaning and syntax as the Helm flags of the same names:
+`--values` specifies a YAML file with custom values and `--set` sets values directly
+on the command-line. Both can be provided multiple times:
+
+```bash
+$ tele build example/app.yaml --values=custom-values.yaml --set=nginx.image=1.9.1 --set=postgres.registry=internal.registry.io
+```
 
 During the installation, the vendored images will be pushed to the Cluster's local
-Docker registry which is available inside the Cluster at `leader.telekube.local:5000`.
+Docker registry which is available inside the Cluster at `registry.local:5000`.
```

Comment:
Is there an easy to discover what docker images are available in the registry.local?

Reply:
If by "easy" you mean calling the appropriate registry API with proper authentication then yes :) But really, no - we have a [ticket](https://github.com/gravitational/gravity/issues/696) in this milestone to provide a convenience script for that.